modules/database/src/template/top/exampleBoot/ioc/README@RTEMS

```diff
@@ -4,3 +4,7 @@ contents to
 
 Then load the executable into the IOC (floppy disk, network boot, debugger,
 etc.) and start it.
+
+
+SPDX-FileCopyrightText: 2001 Lawrence Berkeley National Laboratory
```

Comment:
In 2001 Eric Norum was working for APS at Argonne, but we converted the EPICS CVS repository to Bazaar and then to Git after he had moved to LBL. The repo conversion put his newer email address on all his commits — CVS only records the local username of the committer, so the repo conversion tools have to be told how to convert them.

Reply:
Done. Thank you so much for reviewing these things!